docs/contributing/community-contributions.md

```diff
@@ -2,22 +2,23 @@
 title: Community Contributions
 ---
 
-A big part of what makes Gatsby great is each and every one of you in the community. Your contributions enrich the Gatsby experience and make it better every day.
+इस समुदाय के मे प्रत्येक जन Gatsby को महान बनाने में एक बड़ा भाग रखता है. आपका योगदान Gatsby अनुभव को समृद्ध करता है और इसे हर दिन बेहतर बनाता है|
 
-We welcome all contributions from you in the community, and would be thrilled to amplify your voice. Contributions are not limited to code, and can take all shapes and forms:
 
-- Your wonderful Gatsby.js website!
-- Gatsby starters
-- Gatsby plugins
-- Blog posts and case studies
-- Talk and lecture videos
-- Workshop materials
-- GitHub issues and pull requests
+हम समुदाय में आप सभी के योगदान का स्वागत करते हैं, और हम आपके आवाज़ को बढ़ने में रोमांच रखते है| योगदान देना सिर्फ कोड तक ही सीमित नहीं है, और यह सब तरह के आकार और रूप ले सकता है जैसे:
```

Comment:
`रोमांच`

does not seems correct here. Any other thoughts?

Reply:
@arshadkazmi42 
How about `रूचि`?

Reply:
this sounds better